en/dronecan/README.md

```diff
@@ -64,50 +66,71 @@ Supported hardware includes (this is not an exhaustive list):
 
+:::note
+The PX4 node ID can be configured using the [UAVCAN_NODE_ID](../advanced_config/parameter_reference.md#UAVCAN_NODE_ID) parameter.
+The parameter is set to 1 by default.
+:::
+
 :::warning
 At time of writing, PX4 does not run the node allocation server on the CAN2 port.
 This means that if you have a device that is *only* connected to CAN2 (not redundantly to CAN1 and CAN2), you will need to manually configure its node ID.
 :::
 
 ## PX4 Configuration
 
+DroneCAN is configured on PX4 by [setting specific PX4 parameters](../advanced_config/parameters.md) in QGroundControl.
+You will need to enable DroneCAN itself, along with subscriptions and publications for any features that you use.
+
+:::note
+In some cases you may need to also configure parameters on the connected CAN devices (these can also be [set using QGC](#qgc-cannode-parameter-configuration)).
+:::
+
 ### Enabling DroneCAN
 
-To enable the DroneCAN driver, set the [UAVCAN_ENABLE](../advanced_config/parameter_reference.md#UAVCAN_ENABLE) parameter:
+To enable the PX4 DroneCAN driver, set the [UAVCAN_ENABLE](../advanced_config/parameter_reference.md#UAVCAN_ENABLE) parameter:
 
 - `0`: DroneCAN driver disabled
-- `1`: DroneCAN driver enabled for sensors, DNA server disabled
+- `1`: DroneCAN driver enabled for sensors, [DNA server](#node-id-allocation) disabled
 - `2`: DroneCAN driver enabled for sensors, DNA server enabled
 - `3`: DroneCAN driver enabled for sensors and ESCs, DNA server enabled
 
 `2` or `3` are recommended, if DNA is supported.
 
+### DroneCan Subscriptions & Publications
+
+PX4 does not publish or subscribe to DroneCAN messages that _might_ be needed by default, in order to avoid spamming the CAN bus.
+Instead you must enable publication or subscription to the messages associated with a particular feature by setting the associated [UAVCAN parameter](../advanced_config/parameter_reference.md#uavcan).
+
+For example, to use a connected DroneCAN smart battery you would enable the [UAVCAN_SUB_BAT](../advanced_config/parameter_reference.md#UAVCAN_SUB_BAT) parameter, which would subscribe PX4 to receive [BatteryInfo](https://dronecan.github.io/Specification/7._List_of_standard_data_types/#batteryinfo) DroneCAN messages.
+If using a peripheral that needs to know if PX4 is armed, you would need to set the [UAVCAN_PUB_ARM](../advanced_config/parameter_reference.md#UAVCAN_PUB_ARM) parameter so that PX4 starts publishing [ArmingStatus](https://dronecan.github.io/Specification/7._List_of_standard_data_types/#armingstatus) messages.
+
+The parameter names are prefixed with `UAVCAN_SUB_` and `UAVCAN_PUB_` to indicate whether they enable PX4 subscribing or publishing.
+The remainder of the name indicates the specific message/feature being set.
+
+DroneCAN peripherals connected to PX4 can also be [configured using via QGC](#qgc-cannode-parameter-configuration).
+These are named with the prefix [CANNODE_](../advanced_config/parameter_reference.md#CANNODE_BITRATE).
```

Comment:
What I think is that CANNODE_ is a prefix - this just tells you that there is some param on a particular GPS CANNODE probably called GPS_RTCM or whatever (not CANNODE_RTCM). So its a hint. However I am still chasing this up in https://discord.com/channels/1022170275984457759/1153875133329571951/1154281387940118538.

Or it may be that the name is literally what the param should be called on the node. I just need to know.

If this is true then I'd add this information to the section below on QGC node connection. I'd also cross link from all the sections that say "set whatever on the node, like CANNODE_TERM.